### PR TITLE
Cranberry Apple Cider Cocktail

### PR DESCRIPTION
**Pull request by nitish03**
Added new recipe Cranberry Apple Cider Cocktail

### CODE REVIEW

File: src/recipes/cranberry-apple-cider-cocktail.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      "measure": "bottle",
+      "ingredient": "hard cider (Angry Orchard/ Stella Artois Cidre)"
+    }
+  ],
+  "directions": [
+    "Fill glass and cocktail shaker with ice",
+    "then Add cranberry juice, orange juice and vodka to the cocktail shaker and shake until chilled. Strain into ice filled glass.",
+    "Top off the glass with the hard cider and garnish with apple slice"
+  ],
+  "image": "Cranberry-Apple-Cider-Cocktail.jpg",
```
Comment (alfg) [inline src/recipes/cranberry-apple-cider-cocktail.json]: Can we keep the filename lowercase?
```suggestion
  "image": "cranberry-apple-cider-cocktail.jpg",
```

Comment (nitish03) [reply src/recipes/cranberry-apple-cider-cocktail.json]: @alfg  I changed in lowercase fonts. thanks!

**Comment by alfg** [comment]
The image filename should also be updated to lowercase. Also, please provide a horizontal image at 600x400px if possible. And revert the CONTRIBUTING.md file so it doesn't appear in this PR. Thanks!

**Comment by alfg** [comment]
Closing. Feel free to re-open with requested fixes.

**Pull request closed without merging**